bin/strata-cli/src/main.rs

```diff
@@ -47,5 +47,6 @@ async fn main() {
         Commands::Receive(args) => receive(args, seed, settings, esplora).await,
         Commands::Reset(args) => reset(args, persister, settings).await,
         Commands::ChangePwd(args) => change_pwd(args, seed, persister).await,
+        Commands::Scan(args) => scan(args, seed, esplora).await,
```

Comment:
We also need a simple `Sync` command as well.
https://docs.rs/bdk_wallet/latest/bdk_wallet/struct.Wallet.html#method.start_sync_with_revealed_spks

Reply:
We can move this to a ticket and do in a separate PR if necessary

Reply:
Ok! Yeah you're right in abstracting that away from the user.